src/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.block1 {
+  flex-basis: 33.3%;
+  flex-grow: 1;
+  background-color: rgb(255,0,0);
+}
+
+.block2 {
+  flex-grow: 1;
+  flex-basis: 33.3%;
+  background-color: rgb(204,0,0);
+}
+
+.block3 {
+  flex-basis: 33.3%;
+  flex-grow: 1;
+  background-color: rgb(153,0,0);
+}
+
+.block4 {
+  flex-basis: 33.3%;
+  flex-grow: 1;
+  background-color: rgb(102,0,0);
+}
+
+.block5 {
+  flex-basis: 33.3%;
+  flex-grow: 1;
+  background-color: rgb(51,0,0);
+}
+
+.block6 {
+  flex-basis: 33.3%;
+  flex-grow: 1;
+  background-color: rgb(0,0,0);
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 100px;
+  font-family: Arial, sans-serif;
+  min-height: 300px;
+  color: white;
+}
+
+@media (min-width: 600px) {
+  .container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    height: 900px;
+  }
```

Comment:
make container as default styles.Media change only order for blocks with modificator like
```
  .block--4 {
     order: 3;
   }
```
and change   flex-basis for all block with class block